python/datasources/graphql_ahr.py

```diff
@@ -105,8 +105,12 @@ def write_to_bq(self, dataset, gcs_bucket, write_local_instead_of_bq=False, **at
             table_name = f"{demographic}_{geo_level}_{time_view}"
             topic_prefixes = [std_col.extract_prefix(rate_col) for rate_col in AHR_BASE_MEASURES_TO_RATES_MAP.values()]
             topic_prefixes.append('ahr')
+            df = df[df['time_period'].astype(int) <= 2022]
```

Comment:
we should avoid "magic numbers" like this hard coded in; instead can you make a variable at the top of the file like `LAST_COMPLETE_DATA_YEAR` or similar, and then use it here? it just helps a lot to name the variable so then we remember why the year is there to begin with, and easier to remember to update it as the data changes.